libs/entitlement-runtime/src/test/resources/org/elasticsearch/entitlement/policy/test-policy.yaml

```diff
@@ -0,0 +1,9 @@
+policy:
+  - module:
+      name: entitlement-test
+      entitlements:
```

Comment:
How do you envision this structure extending to support class-level grants?

Reply:
I envisioned having sub objects of module for package and then for class with their own set of entitlements for those specific objects. @rjernst and I had chatted about having all scoping be at the top-level where scope was separate by a colon, but I thought the trade off of having them be sub objects made it easier for users to see what permissions belong where. I'm really open to changing this either way.

Reply:
Oh I think I get it. This is the hierarchical structure you mentioned. I like it!

Reply:
I think it makes sense. We can certainly make changes to simplify or move structure around if we want to.